website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Data Lake Analytics: Alibaba's Federated Cloud Strategy
+author: George Wang
+authorURL: https://www.linkedin.com/in/george-wang-9a5a46190/
+---
+
+
+## Data Federation Service
+
+Today’s cloud database empowers cloud users to easily and directly interact with other cloud products and services, available to be built for their own cloud data lake analytic scenarios and solutions. The architecture to pile up a cloud-based data lake analytic scenario is divided into three layers:
+
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/threeLayerCloud_v1.png)
+<br><br>
+**First Layer**: A variety of selected cloud native data storage and database services. 
+<br><br>
+Users store and place their data on a series of cloud-native data storage services and cloud databases systems. Cloud data sources vary depending on different business use cases and business scenarios, but in general speaking, there are three cloud data storage: NoSQL and relational databases. Various choices of storage service may include: cost-effective object cloud storage which may store files, structured, semi-structured, unstructured raw data, multimedia files; NoSQL, RDS, and other cloud database services;
+<br><br>
+
+**Second Layer**: Cloud native data lake analysis service layer.
+<br><br>
+This layer emphasizes a very important feature in Cloud Native computing, that is, Serverless is server-free and is the basic service for building SaaS on the cloud, except for Data Lake analysis scenarios, there is also a more ubiquitous serverless PaaS and Function-computing as-a-Service (FaaS) such as Alibaba Cloud Function Calculation. This article focuses on the serverless analysis capability of Data Lake analytic scenarios.
+<br>
+1. Elasticity- which allows flexibility capability meaning on demand, flexible in time, predictable and intelligent mixed load handling ability
+2. Resiliency - User’s unperceived High Availability capability, including transparent rolling upgrade, fast handover, disaster tolerance across AZ (available area), etc
```

Comment:
Maybe shorten and rephrase this to

"High Availability capability, including transparent rolling upgrade, fast failover, and disaster tolerance across AZ (available area), etc"

Reply:
so for this I'd like to revised a bit by putting the same pattern because it should be clear to user what are the key component features offered in this second layer